README.md

```diff
@@ -50,6 +50,8 @@ Run the simulator:
 poetry run python examples/run.py
 ```
 
+Run from the top-level of the repository.
```

Comment:
Nice, thank you for adding this note. Improving documentation is always a good thing to do as we work!